internal/services/devcenter/dev_center_dev_box_definition_resource.go

```diff
@@ -161,6 +175,10 @@ func (r DevCenterDevBoxDefinitionResource) Read() sdk.ResourceFunc {
 						state.ImageReferenceId = pointer.From(v.Id)
 					}
 
+					if v := props.HibernateSupport; v != nil {
+						state.HibernateSupport = !strings.EqualFold(string(pointer.From(v)), string(devboxdefinitions.HibernateSupportDisabled))
+					}
```

Comment:
Same suggestion as my comment on the data source

```suggestion
					state.HibernateSupport = pointer.From(props.HibernateSupport) == devboxdefinitions.HibernateSupportEnabled
```

Reply:
Same as well.